### PR TITLE
Update converter to fix digest bug

### PR DESCRIPTION
**Pull request by moria97**

### CODE REVIEW

**Review by hongyh13 — APPROVED** [review]

File: FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.HealthCheck/Checkers/AzureContainerRegistryHealthChecker.cs [hunk 1]
```diff
@@ -19,7 +19,7 @@ namespace Microsoft.Health.Fhir.Synapse.HealthCheck.Checkers
 {
     public class AzureContainerRegistryHealthChecker : BaseHealthChecker
     {
-        private const string MediatypeV2Manifest = "application/vnd.docker.distribution.manifest.v2+json";
+        private const string ManifestMediatype = "application/vnd.oci.image.manifest.v1+json,application/vnd.docker.distribution.manifest.v2+json";
```
Comment (sowu880) [inline FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.HealthCheck/Checkers/AzureContainerRegistryHealthChecker.cs:22]: Add two variables here? and add reference to converter?

Comment (moria97) [reply FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.HealthCheck/Checkers/AzureContainerRegistryHealthChecker.cs:22]: This is for hotfix. We can refine this later

Comment (sowu880) [inline FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.HealthCheck/Checkers/AzureContainerRegistryHealthChecker.cs:22]: Also need to update it in Microsoft.Health.Fhir.Synapse.Core/Jobs/ExternalDependencyChecker.cs

Comment (moria97) [reply FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.HealthCheck/Checkers/AzureContainerRegistryHealthChecker.cs:22]: Nice catch. Will fix it

**Review by sowu880 — APPROVED** [review]

**Pull request merged**